src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -468,7 +468,7 @@ const EnvironmentPicker = ({
         resources: Array<IAzureResource | string>
     ) => {
```

Comment:
Did we give types to the local storage read/write operations?

Reply:
yes